### PR TITLE
deps(client): Add missing dependencies

### PR DESCRIPTION
**Pull request by tylerbutler**
Adds missing dependencies that were found while investigating switching to pnpm.

### CODE REVIEW

File: packages/tools/client-debugger/client-debugger/package.json [hunk 1]
```diff
@@ -75,13 +75,15 @@
...
     "@types/chai": "^4.0.0",
+    "@types/events": "^3.0.0",
     "@types/mocha": "^9.1.1",
     "chai": "^4.2.0",
     "concurrently": "^6.2.0",
     "copyfiles": "^2.4.1",
     "eslint": "~8.6.0",
     "eslint-config-prettier": "~8.5.0",
     "eslint-plugin-chai-expect": "~3.0.0",
+    "events": "^3.1.0",
```
Comment (Josmithr) [inline packages/tools/client-debugger/client-debugger/package.json:86]: Should this not be a proper dependency? Or does this only get used in the tests?

**Review by Josmithr — APPROVED** [review]

File: packages/tools/client-debugger/client-debugger/package.json (hunk 1 above)
Comment (Josmithr) [reply packages/tools/client-debugger/client-debugger/package.json:86]: Actually, I don't see "events" being used in this package at all. Our shared `IEventProvider` and `TypedEventEmitter` types are used, but "events is not used directly. Are these needed as a "peer" of `common-utils`?

Comment (tylerbutler) [reply packages/tools/client-debugger/client-debugger/package.json:86]: @Josmithr EventEmitter is used in the tests:

https://github.com/microsoft/FluidFramework/blob/73a8c288ce1743057b49955049ff0e0087ef5058/packages/tools/client-debugger/client-debugger/src/test/Utilities.ts#L6

You're right that this should be in deps, and there should also be a types package. Fixing.

Comment (Josmithr) [reply packages/tools/client-debugger/client-debugger/package.json:86]: Oh, my bad. My branch was farther behind than I thought. Looks good then!

**Pull request merged**